### ISSUE TITLE
Inventory State - Drop and Pick Up of items

### ISSUE THREAD

**Issue by phobos2077** [falltergeist/falltergeist#435]
* Drop items using context menu in inventory.
* Show item count popup when number of items in stack is > 1.
* Pickup items from ground - play "magic hands ground" animation and sound effect, different for each item type.
* Check for current carry capacity when picking up and display message in log when dude can't lift anymore (always allow to pick up 0-weight items, like drugs and money).

Note:
Original behavior was that when you drop 5 items to the ground, you can only pick them up one-by-one (click 5 times).

**Comment by alexeevdv** [comment]
We need to reconsider original behavior. Maybe similar items(like cups) needs to be grouped

**Comment by phobos2077** [comment]
I agree. Such things can be corrected and if, in future, people will demand original behavior, someone can always come in and implement it with option to turn on/off :)
